pkg/ccl/backupccl/backup_destination.go

```diff
@@ -120,13 +129,44 @@ func resolveDest(
 			return "", "", "", nil, nil, err
 		}
 		if exists {
-			// The backup in the auto-append directory is the full backup.
+			// The defaultStore contains a full backup; consequently,
+			// we're conducting an incremental backup.
 			prevBackupURIs = append(prevBackupURIs, defaultURI)
-			priors, err := FindPriorBackups(ctx, defaultStore, OmitManifest)
+
+			var priors []string
+			var backupChainURI string
+			if len(incrementalStorage) > 0 {
+				// Implies the incremental backup chain lives in
+				// incrementalStorage/chosenSuffix, while the full backup lives in
+				// defaultStore/chosenSuffix. The incremental backup chain in
+				// incrementalStorage/chosenSuffix will not contain any incremental backups that live
+				// elsewhere.
+				_, backupChainURI, err = getLocalityAndBaseURI(incrementalStorage[0],
```

Comment:
that is special about [0]?

Reply:
that is, should we be using `getURIsByLocalityKV` and then using its defaultURI return here instead of [0] in case the default bucket isn't in position 0?

Reply:
Done